.github/workflows/next-binaries.yml

```diff
@@ -0,0 +1,65 @@
+name: Publish Nightly Next Binaries
+on:
+  schedule:
+    - cron: '10 3 * * *'
```

Comment:
3:10 AM everyday?

Reply:
GMTsee docs here https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule